.github/workflows/ci.yml

```diff
@@ -51,6 +51,14 @@ jobs:
           make clean && make doctest
         working-directory: docs/
 
+      - name: Ensure docs build - PyVortex
+        run: |
+          set -ex
+
+          (cd pyvortex && uv run maturin develop)
+
+          cd docs
+          uv run sphinx-build -M html . _build --fail-on-warning --keep-going
```

Comment:
can we just use the makefile here?

Reply:
Sure, I can do that in a follow up PR since this already slipped in.

Reply:
https://github.com/spiraldb/vortex/pull/1156